steps/variables-libvirt.tf

```diff
@@ -53,12 +53,30 @@ variable "tectonic_libvirt_etcd_memory" {
 
 variable "tectonic_libvirt_master_memory" {
   type        = "string"
-  description = "ram to allocate for each etcd node"
+  description = "ram to allocate for each master node"
   default     = "2048"
 }
 
 variable "tectonic_libvirt_worker_memory" {
   type        = "string"
-  description = "ram to allocate for each etcd node"
+  description = "ram to allocate for each worker node"
   default     = "1024"
 }
+
+variable "tectonic_libvirt_etcd_vcpu" {
```

Comment:
nit: I realize there's an existing `tectonic_*` pattern, but since we're trying to rebrand this from the Tectonic installer to the OpenShift installer, I'd rather have new variables land without a prefix.  That would make this `libvirt_etcd_vcpu`.  But we'll obviously have to go through and drop `tectonic_` prefixes from our existing entries at some point, and dropping it from this variable too won't be that much more work, so I'm fine leaving it as you have it if you prefer the `tectonic_` prefix now.